pkg/controller/mizar/util.go

```diff
@@ -107,20 +149,213 @@ func ConvertToNodeContract(node *v1.Node) *BuiltinsNodeMessage {
 		}
 	}
 
-	klog.V(3).Infof("Node Name: %s, IP: %s", node.Name, ip)
+	klog.Infof("Node Name: %s, IP: %s", node.Name, ip)
 	return &BuiltinsNodeMessage{
 		Name: node.Name,
 		Ip:   ip,
 	}
 }
 
 func ConvertToNetworkPolicyContract(policy *networking.NetworkPolicy) *BuiltinsNetworkPolicyMessage {
-	klog.V(3).Infof("NetworkPolicy Name: %s, Namespace: %s, Tenant: %s",
+	klog.Infof("NetworkPolicy Name: %s, Namespace: %s, Tenant: %s",
 		policy.Name, policy.Namespace, policy.Tenant)
+	policyJson, _ := json.Marshal(parseNetworkPolicySpecToMsg(policy.Spec))
+	klog.Infof("Policy: %s", string(policyJson))
 
 	return &BuiltinsNetworkPolicyMessage{
 		Name:      policy.Name,
 		Namespace: policy.Namespace,
 		Tenant:    policy.Tenant,
+		Policy:    string(policyJson),
+	}
+}
+
+func parseNetworkPolicySpecToMsg(nps networking.NetworkPolicySpec) MizarNetworkPolicyPolicySpecMsg {
```

Comment:
This function can be simplifed as below ( most local variables can be skipped):
{
	return MizarNetworkPolicyPolicySpecMsg{
		PodSel: MizarNetworkPolicyPodSelector{
		     MatchLabels: nps.PodSelector.MatchLabels,
	        }
		In:     parseNetworkPolicyIngressRulesToMsg(nps.Ingress),
		Out:    parseNetworkPolicyEgressRulesToMsg(nps.Egress),
		Type:   policyTypesToStringArray(nps.PolicyTypes),
	}
}